### DISCUSSION TITLE
LiveCanvas?

### DISCUSSION THREAD

**Discussion by Schiehsl3** [bootscore/bootscore#311] [Q&A]
I am trying to use LiveCanvas which I did with no problem in previous versions of bootscore. I seem to be running into a lot of styling issues with the most recent version of either bootscore or livecanvas not sure which. 

I noticed the landing page for livecanvas was removed from the bootscore website at some point. Is there a reason for this? Is it no longer compatible?

**Comment by crftwrk** [comment]
Hi,

LC developer did not response to any inquiry, so we decided to remove the demo page.

However, problem should be solved quickly. In version 5.2 we moved the page templates to a subfolder. As far as I can remember, LiveCanvas uses a specific page template in root folder.

- Go in main-theme to folder `page-templates`
- Copy the file with the funny name `page-blank-without-container.php` direct to your child's root folder

Let me know if that helps.

**Comment by Schiehsl3** [comment]
When I attempted to email them pre-sales only the first few made it to my inbox in replies. They showed me a few month later when I tried them at another email that they had been replying but somehow their ticket system wasn't sending me the emails. 

This might be the case for you as well. I will try your fix later when I get home from work. Thanks :D

**Comment by Schiehsl3** [comment]
I just tried your idea, it didn't work. 

https://library.livecanvas.com/starters/homepage-software/?screenshot this is how the page should look. 

I have included how it looks. 

Its like the spacing and sizes is all wrong. I works fine with picostrap though. 

![screencapture-aliaswebs-sap-2022-11-08-14_49_17](https://user-images.githubusercontent.com/1226464/200661246-1510c41b-7f19-4118-ba94-ba64791f25c5.png)
![screencapture-aliaswebs-sap-2022-11-08-14_48_55](https://user-images.githubusercontent.com/1226464/200661248-e952ef4c-d6f7-4797-bb8c-b0b3027924e3.png)

**Reply by crftwrk** [reply]
We're in contact with LC developer. I'll keep you updated.

**Reply by Schiehsl3** [reply]
sounds good, might be best to use their ticket system in case their email fails again :)

**Comment by Schiehsl3** [comment]
Any updates on this?

**Reply by crftwrk** [reply]
LC developer said that he is in contact with you, isn‘t he?

**Reply by Schiehsl3** [reply]
Yes

**Reply by crftwrk** [reply]
Is he in contact with you or not? He wrote that bug has been fixed but I have no further update yet.

**Reply by Schiehsl3** [reply]
he told me of a fix but I was writing back because I didn't know if there was any update on bootscore itself instead of having to do workarounds to make it work.

**Reply by crftwrk** [reply]
Latest is v5.2.2.1 https://github.com/bootscore/bootscore/releases

Changing the structure of page templates was in v5.2.0.0, guess LC breaking was in this release. 5.2.2.1 has not any breaking changes to 5.2.0.0. So, simply update to latest version. What version do you use and works LC fine now?
